src/views/components/authentication/form/AuthenticationForm.js

```diff
@@ -82,6 +82,7 @@ class AuthenticationForm extends Component {
         onSubmit={this.handleSubmit}
         validateOnBlur
         enableReinitialize
+        testID="authenticationForm"
```

Comment:
Likewise for the rest of the `testID`s in this PR, they should point to a prop instead of being hardcoded.